vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -7,15 +7,15 @@ sidebarDepth: 2
 This page provides an overview of the key Entando GitHub repositories with brief descriptions 
 of how these repositories are realized in a running Entando deployment. It also explores the architecture behind the Entando Platform and how its functions are structured.
 
-## entando-operator
-The Entando Operator coordinates the installation and configuration of all the components in an Entando 
-cluster. The operator can be installed once per cluster and is used to coordinate the plugin lifecycle for multiple Entando Applications across many namespaces.
+## Entando Operator
+The **Entando Operator** coordinates the installation and configuration of all the components in an Entando 
+cluster. The operator can be installed once per cluster, and is used to coordinate the plugin lifecycle for multiple Entando Applications across many namespaces.
```

Comment:
Ouch, this description is wrong now. We don't currently support cluster-level installation of the operator. Change to "The **Entando Operator** coordinates the installation and configuration of all the components of an Entando Application."

Reply:
but then is the second sentence wrong, that the operator can't be installed once per cluster and does not coordinate the plugin lifecycle for multiple Entando Apps across many namespaces?

Reply:
yes. per conversation today.